lib/_http_outgoing.js

```diff
@@ -908,6 +908,10 @@ OutgoingMessage.prototype.write = function write(chunk, encoding, callback) {
 };
 
 function onError(msg, err, callback) {
+  if (msg.destroyed) {
+    return;
```

Comment:
Does this mean that if I call `end(chunk, CB)` and `this` is both `finished` and `destroyed`, the callback won't be called?